index.html

```diff
@@ -57,12 +58,33 @@ <h1 class="font-header header-padding margin-zero">New Entry</h1>
           </div>
           <div class="row">
             <div class="column-full display-flex row-reverse">
-              <button class="purple-background font-body button" type="submit">SAVE</button>
+              <button class="purple-background font-body button" data-view="entry-form" type="submit">SAVE</button>
             </div>
           </div>
         </div>
       </form>
     </div>
+    <div class="view" data-view="entries">
+      <div class="container font-body">
+        <div class="row">
+          <div class="column-full display-flex justify-between">
+            <h1 class="font-header header-padding margin-zero">Entries</h1>
+            <a href="#" id="new-link" data-view="entry-form" class="display-flex align-center purple-background font-body button justify-center margin-auto link">NEW</a>
+          </div>
+        </div>
+        <ul class="padding-zero">
+        </ul>
+      </div>
+    </div>
+    <div class="" id="no-entry" data-view="no-entries">
+      <div class="container font-body">
+        <div class="row">
+          <div class="column-full display-flex justify-center align-center">
+            <p >No entries have been recorded.</p>
+          </div>
+        </div>
+      </div>
+    </div>
```

Comment:
This shouldn't be its own separate view, it can just be an element within the `entries` view that is hidden or shown based on if there are any entries.